docs/guide/index.asciidoc

```diff
@@ -251,5 +251,60 @@ pip install elastic-apm
 The Python agent supports Django and Flask out of the box.
 See the {apm-py-ref}/index.html[APM Python Agent documentation] for more details.
 
+[[ruby-agent]]
+[float]
+==== Install the Ruby agent
+
+Add the gem to your Gemfile:
+
+[source,bash]
+----------------------------------
+gem 'elastic-apm'
+----------------------------------
+
+Create a config file config/elastic_apm.yml:
+
+[source,bash]
+----------------------------------
+server_url: http://localhost:8100
+secret_token: ''
+----------------------------------
+
+The Ruby agent automatically instruments Rails out of the box.
```

Comment:
@simitt Rack is also auto-instrumented?

Reply:
some more setup necessary when not Rails https://www.elastic.co/guide/en/apm/agent/ruby/current/getting-started-rack.html